src/midi/fluid_seq_queue.cpp

```diff
@@ -192,16 +192,15 @@ static void fluid_seq_queue_pop(seq_queue_t &queue)
     queue.pop_back();
 }
 
-void fluid_seq_queue_process(void *que, fluid_sequencer_t *seq)
+void fluid_seq_queue_process(void *que, fluid_sequencer_t *seq, int cur_ticks)
 {
     seq_queue_t& queue = *static_cast<seq_queue_t*>(que);
 
-    unsigned int ticks = fluid_sequencer_get_tick(seq);
     while(!queue.empty())
     {
         // Get the top most event.
         const fluid_event_t& top = queue.front();
-        if(top.time <= fluid_sequencer_get_tick(seq))
+        if(top.time <= cur_ticks)
```

Comment:
Suggestion to avoid warning.
        if(top.time <= `(unsigned int)`cur_ticks)

Reply:
You're right. But a cast is not correct here. `cur_ticks` should be unsigned instead. I'll fix it.